package.json

```
@@ -99,6 +99,7 @@
     "loggerhythm": "~3.0.3",
     "moment": "~2.24.0",
     "node-uuid": "^1.4.8",
+    "pg": "^7.9.0",
```

You also need to add `pg-hstore`, or Sequelize won't be able to use postgres.
See:
http://docs.sequelizejs.com/manual/getting-started

I'd also ask that you use `~` with these packages, because I want to test minor version bumps before implementing them.

Given the kind of nasty surprises we had with some packages in the past, I'd like to be on the safe side with critical packages such as these.

```suggestion
    "pg": "~7.9.0",
    "pg-hstore": "~2.3.2",
```

---

Do you also plan to implement postgres tests for jenkins? 

Because I don't want to officially include postgres with the runtime, until we can be sure that all the tests are working.